cwl/src/test/scala/cwl/CommandLineToolSpec.scala

```diff
@@ -0,0 +1,23 @@
+package cwl
+
+import cwl.CommandLineTool.CommandInputParameter
+import org.scalatest.{FlatSpec, Matchers}
+
+class CommandLineToolSpec extends FlatSpec with Matchers {
+
+  behavior of "CommandLineTool"
+
+  it should "order input arguments when bound to command line" in {
+
+    val a = CommandInputParameter("a", inputBinding = None)
```

Comment:
OOI, what is this case where `inputBinding` is `None`?

Reply:
An element could be used in an expression and not included in the command line script.